### DISCUSSION TITLE
How to turn off auto fill on v-text

### DISCUSSION THREAD

**Discussion by behrouzbk** [vuetifyjs/vuetify#13662] [General] [answered]
Have email and password in register form and want to not be filled automatically by bowser. please help and let me know

` <v-col  md="6" sm="12">
                                        <v-text-field
                                                label="Email"
                                                v-model="email"
                                                :rules="[rules.required, rules.email]"
                                                type="email"
                                                error-count="2"
                                                aria-required="true"
                                                class="required"
                                                required
                                                dense
                                        />
                                    </v-col>
                                    <v-col  md="6" sm="12">
                                        <v-text-field
                                                v-model="password"
                                                :append-icon="showPass ? 'mdi-eye' : 'mdi-eye-off'"
                                                :rules="passwordRules"
                                                :type="showPass ? 'text' : 'password'"
                                                label="Password"
                                                hint="At least 8 characters"
                                                @click:append="showPass = !showPass"
                                                aria-required="true"
                                                class="required"
                                                required
                                                dense
                                        />
                                    </v-col>`

**Comment by Maxim-Mazurok** [comment, marked as answer]
Check out https://github.com/vuetifyjs/vuetify/issues/2792
